### ISSUE TITLE
Can't load refractive material with k values

### ISSUE THREAD

**Issue by MLackner** [stillyslalom/RefractiveIndex.jl#12]
```julia
julia> RefractiveMaterial("main", "Pt", "Werner")
ERROR: KeyError: key :coefficients not found
Stacktrace:
 [1] getindex
   @ ./dict.jl:482 [inlined]
 [2] RefractiveIndex.DispersionFormula(data::Dict{Symbol, Any})
   @ RefractiveIndex ~/.julia/packages/RefractiveIndex/lOhEM/src/RefractiveIndex.jl:50
 [3] RefractiveMaterial(shelf::String, book::String, page::String)
   @ RefractiveIndex ~/.julia/packages/RefractiveIndex/lOhEM/src/RefractiveIndex.jl:85
 [4] top-level scope
   @ REPL[1]:1
```

This happens for every material I tried that has values for k.

**Comment by stillyslalom** [comment]
Oops, I fixed this back in March, but forgot to tag a new version. Should be fixed once v0.1.3 goes into the registry (https://github.com/JuliaRegistries/General/pull/41016).

**Comment by stillyslalom** [comment]
Fixed in https://github.com/stillyslalom/RefractiveIndex.jl/commit/832050abd74a5178ac6b55a0e3d06d5aa118bbca. You should be able to access `k` values via `(::RefractiveMaterial).dispersion.k(λ [μm])`. I'll probably tweak the API in the next breaking release to return a tuple `(n = ..., k = ...)` via the standard callable-struct method for materials with tabulated `n` and `k`. Let me know if there are any further issues.
